rosetta/services/account.go

```diff
@@ -76,6 +87,46 @@ func (s *AccountAPI) AccountBalance(
 	}, nil
 }
 
+// getStakingBalance used for get delegated balance with sub account identifier
+func (s *AccountAPI) getStakingBalance(
+	subAccount *types.SubAccountIdentifier, addr ethCommon.Address, block *hmyTypes.Block,
+) (*big.Int, *types.Error) {
+	balance := new(big.Int)
+	ty, exist := subAccount.Metadata["type"]
+	if exist {
```

Comment:
Do you mind try handle not exist first please? Like

```
if !exist {
     return nil, error
}
switch ty.(string) {
...
}
```

This will save a bunch of tabs for the following code and make it more readable. Thanks

Reply:
Okie, good idea.